luigi/parameter.py

```diff
@@ -348,21 +348,18 @@ def parse(self, s):
         """Parses a ``float`` from the string using ``float()``."""
         return float(s)
 
-class BooleanParameter(Parameter):
+class BoolParameter(Parameter):
     """A Parameter whose value is a ``bool``."""
-    # TODO(erikbern): why do we call this "boolean" instead of "bool"?
-    # The integer parameter is called "int" so calling this "bool" would be
-    # more consistent, especially given the Python type names.
     def __init__(self, *args, **kwargs):
         """This constructor passes along args and kwargs to ctor for :py:class:`Parameter` but
         specifies ``is_boolean=True``.
         """
-        super(BooleanParameter, self).__init__(*args, is_boolean=True, **kwargs)
+        super(BoolParameter, self).__init__(*args, is_boolean=True, **kwargs)
 
     def parse(self, s):
         """Parses a ``boolean`` from the string, matching 'true' or 'false' ignoring case."""
         return {'true': True, 'false': False}[str(s).lower()]
-
+BooleanParameter = BoolParameter
```

Comment:
Yeah I think that's a great point. We should do that before we merge this commit. Then we can probably drop `BooleanParameter` sometimes in 6-9 months :)